src/main/java/com/synopsys/integration/alert/web/security/authentication/ldap/LdapManager.java

```diff
@@ -86,7 +83,9 @@ public LdapAuthenticationProvider getAuthenticationProvider() throws AlertLDAPCo
 
     public void updateContext() throws AlertLDAPConfigurationException {
         try {
-            if (isLdapEnabled()) {
+            if (!isLdapEnabled()) {
```

Comment:
If we're gonna use the else statement, we can go back to the old way. I just wanted to avoid nesting when there was no other code that executed otherwise. Sorry for the confusion.